### PR TITLE
ORCID Social Identity Provider Extension

### PR DESCRIPTION
**Pull request by cgeorgilakis**

### CODE REVIEW

File: extensions/orcid.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "ORCID Social Identity Provider",
+    "description": "This [Keycloak](https://www.keycloak.org) plugin makes possible to add [ORCID](https://orcid.org/) as Social Identity Provider.",
+        "cgeorgilakis", "laskasn"
+    ],
```
Comment (stianst) [inline extensions/orcid.json:5]:
```suggestion
    "description": "Enables <a href=\"https://orcid.org/\">ORCID</a> as an Identity Provider.",
    "maintainers": [
        "cgeorgilakis", "laskasn"
    ],
```

**Review by stianst — CHANGES_REQUESTED** [review]

File: extensions/orcid.json (hunk 1 above)
Comment (cgeorgilakis) [reply extensions/orcid.json]: Sorry, by mistake. I have corrected it.

Comment (stianst) [reply extensions/orcid.json]: Could you also change the description? It doesn't allow markdown in the description, and I don't think you need to write "Keycloak plugin" as folks are after all seeing this on a list of Keycloak plugins on the Keycloak website ;)

**Review by stianst — APPROVED** [review]

**Comment by stianst** [comment]
Related to https://github.com/keycloak/keycloak/discussions/8620

File: extensions/orcid.json (hunk 1 above)
Comment (cgeorgilakis) [reply extensions/orcid.json]: Done.

**Pull request merged**